src/localplayer.h

```diff
@@ -108,6 +109,9 @@ class LocalPlayer : public Player
 	float hurt_tilt_timer = 0.0f;
 	float hurt_tilt_strength = 0.0f;
 
+	bool autojump = false;
+	float autojump_time = 0.0f;
```

Comment:
These variables should be either private or protected, add getter/setter methods when it's needed outside of `Player` or inherited classes.

Reply:
True; changed. There's a getter now (see `game.cpp`), no use for a setter yet.